### PR TITLE
Fixes infinitylib and outdated config issues

### PR DESCRIPTION
**Pull request by Riley31415**
Updated infinitylib, called function to reset the users config if it is out of date, cleaned up StringRegistry a bit

### CODE REVIEW

File: src/main/java/io/github/seggan/sfcalc/StringRegistry.java [hunk 1]
```diff
@@ -25,18 +27,23 @@
...
-        noItemString = SFCalc.inst().getConfig().getString("no-item-string", "&cThat item was not found.");
-        notANumberString = SFCalc.inst().getConfig().getString("not-a-number-string", "&cThat's not a number!");
-        tooManyCategoriesString = SFCalc.inst().getConfig().getString("category-error-string", "&cThat many categories is not supported yet. Please use the command form of the calculator.");
-        tooManyItemsString = SFCalc.inst().getConfig().getString("item-error-string", "&cThat many items is not supported yet. Please use the command form of the calculator.");
-        notAPlayerString = SFCalc.inst().getConfig().getString("not-a-player-string", "&cYou must be a player to send this message!");
-        invalidNumberString = SFCalc.inst().getConfig().getString("invalid-number-string", "&cInvalid number!");
+    StringRegistry(AddonConfig config) {
+        if (config.getString("header-string").contains("%s")) {
+            // The config is outdated, overwrite it with default values
+            config.resetToDefaults();
```
Comment (WalshyDev) [inline src/main/java/io/github/seggan/sfcalc/StringRegistry.java]: Oh god, please don't just forcefully reset things. You should never force shit for users.

**Review by WalshyDev — CHANGES_REQUESTED** [review]

File: src/main/java/io/github/seggan/sfcalc/StringRegistry.java (hunk 1 above)
Comment (Seggan) [reply src/main/java/io/github/seggan/sfcalc/StringRegistry.java]: agree. we should just let the admins know

**Review by Seggan — APPROVED** [review]
Other than Walshy's thing, looks good.

File: src/main/java/io/github/seggan/sfcalc/StringRegistry.java (hunk 1 above)
Comment (Riley31415) [reply src/main/java/io/github/seggan/sfcalc/StringRegistry.java]: the plugin just doesnt work without resetting it...

Comment (Seggan) [reply src/main/java/io/github/seggan/sfcalc/StringRegistry.java]: i need to monitor that. i havent gotten any complaints in a while so its safe to assume that most servers have switched and we dont need that

**Review by Seggan — APPROVED** [review]

**Pull request merged**